test/functional/services/monaco_editor.ts

```diff
@@ -13,16 +13,19 @@ export function MonacoEditorProvider({ getService }: FtrProviderContext) {
   const browser = getService('browser');
 
   return new (class MonacoEditor {
-    public async getCodeEditorValue() {
-      let request: string = '';
+    public async getCodeEditorValue(index: number = 0) {
```

Comment:
you promised to rename 'index' to more obvious name

Reply:
done